packages/core/integration-tests/test/fs.js

```diff
@@ -81,15 +93,15 @@ describe('fs', function() {
       assert.equal(output, 'hello');
     });
 
-    it('should inline a file with fs require assignment', async function() {
+    it.skip('should inline a file with fs require assignment', async function() {
```

Comment:
should this (and below) be skipped?

Reply:
yeah. this isn't actually safe to inline:

```js
let fs;
fs = require('foo');
```

We don't know what `fs` is at the time that it is used. So now we only handle constant variable declarations rather than assignment expressions.